core/templates/components/rubrics-editor/rubrics-editor.component.ts

```diff
@@ -95,6 +95,39 @@ export class RubricsEditorComponent {
     return Boolean(this.editableExplanations[difficulty][index]);
   }
 
+  ismediumLevelExplanationLengthEqualsOne(difficulty: string): boolean {
+    if (difficulty === this.skillDifficultyMedium) {
+      return this.rubrics[1]._explanations.length === 1;
+    }
+    return false;
+  }
+
+  isMediumLevelExplanationValid(): boolean {
+    if (this.rubrics[1]._explanations.length > 1) {
+      return true;
+    }
+    return this.editableExplanations[this.skillDifficultyMedium][0].length > 0;
+  }
+
+  isTotalExplanationsLengthReached(): boolean {
+    if (this.rubrics[0] === undefined || this.rubrics[1] === undefined ||
+      this.rubrics[2] === undefined) {
+      return false;
```

Comment:
what if `this.rubrics[0] = undefined`, `this.rubrics[1] = 4`, and `this.rubrics[2] = 6`. Then the total is 10 and we should `return true`, right?

Reply:
Umm, please correct me if I'm wrong like the value `undefned` is assigned to variable only at the time of initialization right? After that incase we don't create explanation it will simply be 0. What I tried was I created total 10 explanation from medium only and it was working fine.
Thanks

Reply:
@EricZLou

Reply:
You're right -- thanks @lkbhitesh07!